hazelcast/src/main/resources/hazelcast-config-5.1.json

```diff
@@ -291,6 +291,11 @@
           "type": "boolean",
           "default": false
         },
+        "data-access-enabled": {
```

Comment:
Please cover this change with schema tests as well (both in a happy path and a failure path test).

Reply:
sure, it's a draft PR and it doesn't contain all the required changes